### PR TITLE
Update release-management.md

### PR DESCRIPTION
**Pull request by yann300**

### CODE REVIEW

File: release-management.md [hunk 1]
```diff
@@ -4,10 +4,31 @@ Release managers are responsible for the release management lifecycle, focusing
 
 # Steps of Release Management
 
+## Pre release planning:
+In this stage, release manager and the team lead will elaborate a plan for the coming release. 
+This should take in account:
+ - the current issues list (a fair amount of time should be taken to go over the github issues).
+ - the current critical bugs.
+ - the current roadmap.
+ - each team member should be contacted in order to get different opinions feedback about what the next release should contains.
```
Comment (Aniket-Engg) [inline release-management.md:13]: should *contain*

File: release-management.md [hunk 2]
```diff
@@ -4,10 +4,31 @@ Release managers are responsible for the release management lifecycle, focusing
...
+
+During this phase the github project has to be filled issues that are going to be addressed.
```
Comment (Aniket-Engg) [inline release-management.md:15]: has to be filled *with the* issues

**Review by ioedeveloper — COMMENTED** [review]
Can you include the procedure for beta-release/testers?

**Comment by yann300** [comment]
> Can you include the procedure for beta-release/testers?

@ioedeveloper let's merge this now, i'll create another one later on.

**Review by ioedeveloper — APPROVED** [review]

**Pull request merged**